src/main/java/com/efficio/fieldbook/web/study/germplasm/StudyEntryTransformer.java

```diff
@@ -144,7 +143,7 @@ public Integer getMaleGid() {
 
 					@Override
 					public String getStockIDs() {
-						return studyEntryDto.getStudyEntryPropertyValue(TermId.STOCKID.getId()).orElse("");
+						return "";
```

Comment:
What is the consequence of returning "" ? Is there any information that we are missing when exporting?

Reply:
Use EMPTY